prime-router/metadata/fhir_mapping/hl7/datatypes/CNN/Practitioner.yml

```diff
@@ -0,0 +1,130 @@
+resourceType: Practitioner
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+identifier:
+  expressionType: nested
+  expressionsMap:
+    value:
+      type: STRING
+      valueOf: CNN.1
+      expressionType: HL7Spec
+    extension_assigningAuthority:
+      generateList: true
+      expressionType: nested
+      vars:
+        cnn9: CNN.9
+        cnn10: CNN.10
+        cnn11: CNN.11
+      condition: $cnn9 NOT_NULL || $cnn10 NOT_NULL || $cnn11 NOT_NULL
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          valueOf: assigning-authority
+        extension:
+          generateList: true
+          expressionType: nested
+          expressions:
+            - expressionType: nested
+              expressionsMap:
+                url:
+                  type: SYSTEM_URL
+                  valueOf: assigning-authority-namespace-id
+                valueString:
+                  type: STRING
+                  expressionType: HL7Spec
+                  valueOf: CNN.9
+            - expressionType: nested
+              expressionsMap:
+                url:
+                  type: SYSTEM_URL
+                  valueOf: assigning-authority-universal-id
+                valueString:
+                  type: STRING
+                  expressionType: HL7Spec
+                  valueOf: CNN.10
+            - expressionType: nested
+              expressionsMap:
+                url:
+                  type: SYSTEM_URL
+                  valueOf: universal-id-type
+                valueCode:
```

Comment:
This extension follows a different naming convention from the other two here. Maybe change to `assigning-authority-universal-id-type`?

Reply:
I don't think there is actually a difference between the two,  I think the universal-id-type comes more from it being data in an `HD` rather being related to `assigning-authority`

Reply:
What do you mean an `HD`?

Reply:
Discussed on a call, we'll keep these consistent, but should favor the extension without `assigning-authority`